### PR TITLE
chore(arm): build arm64 snaphot controller and provisioner

### LINKED ISSUE #2991: upgrade the apline image version for openebs to 3.10.3
Some of the openebs containers are using an older version of alpine which is no longer supported. 

Running a trivy check shows the following. 
```
2020-04-08T17:21:30.686Z        WARN    This OS version is no longer supported by the distribution: alpine 3.6.5
```
The containers using older version are:

- quay.io/openebs/m-apiserver:1.9.0-RC1 (alpine 3.6.5)
- quay.io/openebs/provisioner-localpv:1.9.0-RC1 (alpine 3.6.5)
- quay.io/openebs/cspc-operator:1.9.0-RC1 (alpine 3.6.5)
- quay.io/openebs/m-upgrade:1.9.0-RC1 (alpine 3.6.5)
- quay.io/openebs/snapshot-controller:1.9.0-RC1 (alpine 3.6.5)
- quay.io/openebs/snapshot-provisioner:1.9.0-RC1 (alpine 3.6.5)
- quay.io/openebs/velero-plugin:1.9.0-RC1 (alpine 3.6.5)

Note: The follow components use the latest versions of alpine, and trivy passes them.
- quay.io/openebs/linux-utils:1.9.0-RC1 uses (alpine 3.10.3)
- quay.io/openebs/cvc-operator:1.9.0-RC1 (alpine 3.10.4)

### PR DESCRIPTION
**Pull request by prateekpandey14**
- use arm64v8/alpine as a base image for arm base snapshot
controller and snapshot-provisioner images
- bump base image to 3.11.5

fixes: openebs/openebs#2991

Signed-off-by: prateekpandey14 <prateek.pandey@openebs.io>

### CODE REVIEW

File: snapshot/Makefile [hunk 1]
```diff
@@ -36,6 +36,18 @@ ifeq ($(VERSION),)
 	VERSION = latest
 endif
 
+ifeq (${BASE_DOCKER_IMAGE_ARM64}, )
+  BASE_DOCKER_IMAGE_ARM64 = "arm64v8/alpine"
```
Comment (kmova) [inline snapshot/Makefile:40]: Is this required? I think depending no the machine on which docker is running, the docker pull will automatically pick up the arm image.

Comment (prateekpandey14) [reply snapshot/Makefile:40]: Yes , it's required. Docker will not pull unless we mention it.

Comment (kmova) [reply snapshot/Makefile:40]: Nope. apline is a multi-arch manifest. Depending on which OS is pulling the image, the correct docker image will be pulled. 

As travis runs from arm64 and arm64 machine, the correct architecture images are pulled within travis.

**Comment by prateekpandey14** [comment]
Closing this PR as don't required to explicitly mention the alpine arm64v8 base image

**Pull request closed without merging**